File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListBox/Appearance/TextWithImagePage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Forms.CompanionKit.Controls.Util;
using Gizmox.WebGUI.Common.Resources;

#endregion

namespace CompanionKit.Controls.ListBox.Appearance
{
    public partial class TextWithImagePage : UserControl
    {
        public TextWithImagePage()
        {
            InitializeComponent();

            //Get ResourceHandle for photo of customer.
            global::Gizmox.WebGUI.Common.Resources.ResourceHandle photoResource = new ImageResourceHandle("32x32.Photo.png");
            //Set objects collection as DataSource for ComboBox.
            this.mobjListBox.DataSource = Gizmox.WebGUI.Forms.CompanionKit.Controls.Util.CustomerData.GetCustomersWithImage(photoResource);

            //Fill TextBoxes with the values of appropriate properties
            this.mobjDisplayTextBox.Text = this.mobjListBox.DisplayMember;
            this.mobjValueTextBox.Text = this.mobjListBox.ValueMember;
            this.mobjImageTextBox.Text = this.mobjListBox.ImageMember;

        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListBox/Appearance/TextWithImagePage.vb
Namespace CompanionKit.Controls.ListBox.Appearance

	Public Class TextWithImagePage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

			'Get ResourceHandle for photo of customer.
            Dim photoResource As Gizmox.WebGUI.Common.Resources.ResourceHandle = New Gizmox.WebGUI.Common.Resources.ImageResourceHandle("32x32.Photo.png")
			'Set objects collection as DataSource for ComboBox.
            Me.mobjListBox.DataSource = CompanionKit.Controls.Utils.CustomerData.GetCustomersWithImage(photoResource)

            'Fill TextBoxes with the values of appropriate properties
            Me.mobjDisplayTextBox.Text = Me.mobjListBox.DisplayMember
            Me.mobjValueTextBox.Text = Me.mobjListBox.ValueMember
            Me.mobjImageTextBox.Text = Me.mobjListBox.ImageMember
		End Sub

    End Class

End Namespace
